src/pages/Login.tsx

```diff
@@ -0,0 +1,89 @@
+// import * as React from 'react';
+
+// export const Login = () => {
+//   return <h1>hoge</h1>;
+// };
+
+import * as React from 'react';
+// import React, { Component } from 'react';
+import { Form, Button, Container, Row, Alert } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+import User from '../components/User';
+import { useState, useEffect } from 'react';
+
+// import { Button } from '@zeit-ui/react'
+import * as H from 'history';
+import { App } from '../../src/App';
+
+type LoginProps = {
+  email: '';
+  password: '';
+  errMessage: '';
+  history: H.History;
+  message: string;
+};
+
+export const Login = (props: LoginProps) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errMessage, setErrMessage] = useState('');
+
+  useEffect(() => {
+    User.set('isLoggedIn', false.toString());
+    console.log('isLoggedIn(Login.tsx):', User.isLoggedIn());
+  }, []);
+  const click = async () => {
```

Comment:
you should rename this function, because this `click` name don't tell us any information.